lessons/templates/partials/navbar.html

```diff
@@ -0,0 +1,27 @@
+<!--Standardized template for any content in a center card -->
+{% extends 'base.html' %}
+{% block body %}
+
+
+    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+        <a class="navbar-brand" style="margin-left:1%" href='{% url 'home' %}'>MSMS</a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarNav">
+            <ul class="navbar-nav">
+                <li class="nav-item active">
+                    <a class="nav-link" href='{% url 'home' %}'>Home</a>
```

Comment:
This is kinda redundant cause the logo already does the same thing, so you could just delete this